images/installer/Dockerfile.rhel7

```diff
@@ -10,46 +10,50 @@ LABEL name="openshift3/ose-ansible" \
       summary="$SUMMARY" \
       description="$DESCRIPTION" \
       url="https://github.com/openshift/openshift-ansible" \
-      io.k8s.display-name="openshift-ansible" \
-      io.k8s.description="$DESCRIPTION" \
-      io.openshift.expose-services="" \
-      io.openshift.tags="openshift,install,upgrade,ansible" \
       com.redhat.component="aos3-installation-docker" \
-      version="v3.6.0" \
-      release="1" \
       architecture="x86_64"
 
 # Playbooks, roles and their dependencies are installed from packages.
-# Unlike in Dockerfile, we don't invoke the 'assemble' script here
-# because all content and dependencies (like 'oc') is already
-# installed via yum.
 USER root
 RUN INSTALL_PKGS="atomic-openshift-utils atomic-openshift-clients python-boto skopeo openssl java-1.8.0-openjdk-headless httpd-tools" && \
     yum repolist > /dev/null && \
     yum-config-manager --enable rhel-7-server-ose-3.6-rpms && \
     yum-config-manager --enable rhel-7-server-rh-common-rpms && \
+    yum install -y --setopt=tsflags=nodocs ansible && \
```

Comment:
I believe `ansible` gets pulled as a dependency - but if we want to install explicitly maybe just add it to `INSTALL_PKGS` above...